src/yunohost/domain.py

```diff
@@ -220,17 +220,20 @@ def domain_remove(auth, domain, force=False):
     msignals.display(m18n.n('domain_deleted'), 'success')
 
 
-def domain_dns_conf(domain):
+def domain_dns_conf(domain, time_to_live):
     """
     Generate DNS configuration for a domain
 
     Keyword argument:
         domain -- Domain name
+        time to live -- Time to live
```

Comment:
I don't understand why the comment is "time to live -- Time to live"; The function's argument is `ttl`,  so I would prefer the comment to be "ttl -- Time to live"